web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -1160,22 +1166,10 @@
                     urlGetFeature = gnGlobalSettings.proxyUrl
                                         + encodeURIComponent(urlGetFeature);
                   }
-
-                  $.ajax({
-                    url: urlGetFeature
-                  })
-                      .done(function(response) {
-                        // TODO: Check WFS exception
-                        vectorSource.addFeatures(vectorFormat.
-                            readFeatures(response));
-                      })
-                      .then(function() {
-                        this.loadingLayer = false;
-                      });
+                  
+                  return urlGetFeature;
```

Comment:
Instead of doing the full ajax call, make OL do this and we just tell OL where the GetFeature is.